.github/styles/cloudflare/BrandNames.yml

```
@@ -0,0 +1,14 @@
+---
+# Warning: cloudflare.BrandNames
+#
+# Suggests correct spelling and capitalization for brand names.
+#
+# For a list of all options, see https://vale.sh/docs/topics/styles/
+
+extends: substitution
+message: "**Warning**: Use correct brand name: '%s' instead of '%s'."
+level: warning
+ignorecase: false
+swap:
+  CloudFlare: Cloudflare
+  (?!cdnjs)([cC][dD][nN]\.?[jJ][sS]): cdnjs
```

Maybe in a follow up PR if not now.

```suggestion
  (?!cdnjs)([cC][dD][nN]\.?[jJ][sS]): cdnjs
  (?!GitHub)([gG]it\s?[hH]ub): GitHub
  (?!GitLab)([gG]it\s?[lL]ab): GitLab
```